pegasus/sites.v3/hourofcode.com/views/school_not_found.haml

```diff
@@ -0,0 +1,23 @@
+#schoolnotfound
+  %label= hoc_s(:school_not_found_heading)
+
+  .form-group
+    %label.control-label{for: "school-name"}= hoc_s(:school_name)
+    %div
+      %input#hoc-school-name.form-control{name: "school_name_s", type: "text", required: true}/
+
+  .form-group
+    %label.control-label{for: "school-city"}= hoc_s(:school_city)
+    %div
+      %input#hoc-school-city.form-control{name: "school_city_s", type: "text", required: true}/
+
+  %state-zip-box{style: "display: inline-block; width: 500px"}
```

Comment:
not necessary for this change, but might be nice to hoist some of this styling out of the file.